### PR TITLE
Make sprites open-source friendly

### PR DESCRIPTION
**Pull request by kyle-he**

### CODE REVIEW

File: helpers/constants.py [hunk 1]
```diff
@@ -262,37 +262,47 @@ class EmojiManager:
...
-            filter(lambda x: x.name == "gift_normal", gguild.emojis)
-        )
-        self.gift_great = next(filter(lambda x: x.name == "gift_great", gguild.emojis))
-        self.gift_ultra = next(filter(lambda x: x.name == "gift_ultra", gguild.emojis))
+            self.gift_normal = next(
+                filter(lambda x: x.name == "gift_normal", gguild.emojis)
+            )
+            self.gift_great = next(filter(lambda x: x.name == "gift_great", gguild.emojis))
+            self.gift_ultra = next(filter(lambda x: x.name == "gift_ultra", gguild.emojis))
+        except:
```
Comment (oliver-ni) [inline helpers/constants.py:300]: except what?

File: helpers/constants.py [hunk 2]
```diff
@@ -262,37 +262,47 @@ class EmojiManager:
...
+            print("emojis off")
+            self.emojiStatus = False
```
Comment (oliver-ni) [inline helpers/constants.py:302]: should be `snake_case` to adhere with convention

Comment (kyle-he) [reply helpers/constants.py]: fixed

File: helpers/constants.py (hunk 1 above)
Comment (kyle-he) [reply helpers/constants.py]: fixed

**Pull request merged**